### ISSUE TITLE
Tests failing for guest redirect due to a recent change in Laravel

### ISSUE THREAD

**Issue by drehimself** [DCzajkowski/auth-tests#31]
### Environment
- Laravel Version: 6.4.0
- PHP Version: 7.3

### Description:

Two tests are failing because the guest middleware for `ResetPasswordController.php` and `ForgotPasswordController.php` was removed in a recent version of Laravel. See [this commit](https://github.com/laravel/laravel/commit/4036f17416549758816894dc52dc54eabcc13914#diff-74e6f2830c471aa19ab059e48d945275).

```
There were 2 failures:

1) Tests\Feature\Auth\ForgotPasswordTest::testUserCannotViewAnEmailPasswordFormWhenAuthenticated
Response status code [200] is not a redirect status code.
Failed asserting that false is true.

/playground-testing-stuff/vendor/laravel/framework/src/Illuminate/Foundation/Testing/TestResponse.php:198
/Applications/MAMP/htdocs/playground-testing-stuff/tests/Feature/Auth/ForgotPasswordTest.php:51

2) Tests\Feature\Auth\ResetPasswordTest::testUserCannotViewAPasswordResetFormWhenAuthenticated
Response status code [200] is not a redirect status code.
Failed asserting that false is true.

/playground-testing-stuff/vendor/laravel/framework/src/Illuminate/Foundation/Testing/TestResponse.php:198
/Applications/MAMP/htdocs/playground-testing-stuff/tests/Feature/Auth/ResetPasswordTest.php:64

```

**Comment by DCzajkowski** [comment]
They are passing for me on the 6.4.0 release.

Here is the version: https://github.com/DCzajkowski/auth-tests/pull/33/checks?check_run_id=276208895#step:4:51
Here is the pass information: https://github.com/DCzajkowski/auth-tests/pull/33/checks?check_run_id=276208895#step:5:10

**Comment by RalphMRivera** [comment]
I installed a clean copy of Laravel 6.4.0 and get the same result as @drehimself 

```
Vagrant Box ☁  laravel [master] ⚡  ./vendor/bin/phpunit
PHPUnit 8.4.1 by Sebastian Bergmann and contributors.

.............F.......................F.....                       43 / 43 (100%)

Time: 1.99 seconds, Memory: 32.00 MB

There were 2 failures:

1) Tests\Feature\Auth\ForgotPasswordTest::user_cannot_view_an_email_password_form_when_authenticated
Response status code [200] is not a redirect status code.
Failed asserting that false is true.

/home/vagrant/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Testing/TestResponse.php:198
/home/vagrant/laravel/tests/Feature/Auth/ForgotPasswordTest.php:53

2) Tests\Feature\Auth\ResetPasswordTest::user_cannot_view_a_password_reset_form_when_authenticated
Response status code [200] is not a redirect status code.
Failed asserting that false is true.

/home/vagrant/laravel/vendor/laravel/framework/src/Illuminate/Foundation/Testing/TestResponse.php:198
/home/vagrant/laravel/tests/Feature/Auth/ResetPasswordTest.php:66

FAILURES!
Tests: 43, Assertions: 187, Failures: 2.```

**Comment by DCzajkowski** [comment]
I'll verify later, but it's very bizzare :/

**Comment by DCzajkowski** [comment]
I have tested and indeed, I had wrong version set up in Github Actions. Fixed in #32. I will release in a sec.

**Comment by DCzajkowski** [comment]
Version [6.4.0](https://github.com/DCzajkowski/auth-tests/releases/tag/6.4.0) was released 🎉
